src/components/ListItem.tsx

```diff
@@ -16,9 +15,14 @@ export const GenericListItemContainer = styled.Pressable`
   border-radius: 2px;
 `
 
-const Container = styled(GenericListItemContainer)<{ pressed: boolean }>`
+const Container = styled(GenericListItemContainer)<{ pressed: boolean; disabled: boolean }>`
   min-height: ${hp('12%')}px;
-  background-color: ${prop => (prop.pressed ? prop.theme.colors.primary : prop.theme.colors.backgroundAccent)};
+  background-color: ${prop => {
+    if (prop.disabled) {
+      return prop.theme.colors.disabled
+    }
+    return prop.pressed ? prop.theme.colors.primary : prop.theme.colors.backgroundAccent
+  }};
   border: 1px solid ${prop => (prop.pressed ? prop.theme.colors.primary : prop.theme.colors.disabled)};
   padding: ${props =>
```

Comment:
optional because not touched:
`padding: ${props => props.theme.spacings.sm};`
not sure why padding left & right are not equal

Reply:
Because we use this list item also for the selection of a module and there we have this ">" icon, which should be more on the right. https://app.zeplin.io/project/5fc7c98a435298126bcfd4ee/screen/5fd7ad2c3cfec77f5c68f10c